flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/cache/BundledCacheLimitPolicy.java

```diff
@@ -54,6 +56,17 @@ public void release(long toReleaseSize) {
         policies.forEach(policy -> policy.release(toReleaseSize));
     }
 
+    @Override
+    public long usedBytes() {
+        return policies.stream().mapToLong(CacheLimitPolicy::usedBytes).max().getAsLong();
+    }
+
+    @Override
+    public void registerCustomizedMetrics(String prefix, MetricGroup metricGroup) {
+        metricGroup.gauge(prefix + ".usedBytes", () -> usedBytes());
```

Comment:
If the `BundledCacheLimitPolicy` is not enabled (only single policy is enabled), then how does this gauge report?

Reply:
👍 I moved this to the file cache.